src/tests/readytorun/crossgen2/Program.cs

```diff
@@ -2228,7 +2228,15 @@ public static int Main(string[] args)
         RunTest("ExplicitlySizedClassTest", ExplicitlySizedClassTest());
         RunTest("GenericLdtokenTest", GenericLdtokenTest());
         RunTest("ArrayLdtokenTests", ArrayLdtokenTests());
-        RunTest("TestGenericMDArrayBehavior", TestGenericMDArrayBehavior());
+
+        // TODO: BUG BUG: allocation of MD arrays is failing on OSX/ARM64
```

Comment:
Is there a GitHub bug we could reference here? Or should we create one?

Reply:
I will create a bug for this and add a link to this comment.

Reply:
https://github.com/dotnet/runtime/issues/62747